src/simularium/VisGeometry.ts

```diff
@@ -1026,9 +782,15 @@ class VisGeometry {
 
             // collect up the meshes that have > 0 instances
             const meshTypes: InstancedMesh[] = [];
-            for (const entry of this.meshRegistry.values()) {
-                meshTypes.push(entry.instances);
-                this.instancedMeshGroup.add(entry.instances.getMesh());
+            for (const entry of this.geometryStore.registry.values()) {
+                const { displayType } = entry;
+                if (displayType !== GeometryDisplayType.PDB) {
```

Comment:
only Meshes, ie not PDB files, have geometry.instances.  Before we had two seperate registries, one for Meshes and one for PDBs with the idea we might have both a mesh representation and a pdb for one agent. But with the new v3 file format, we're really saying that there is only one geometry per agent type, so the duplicate registry seemed unnecessary. However, in the few cases where we have to do something only to one geometry type, we need to filter the registry down. I thought about making a `geometryStore.getMeshes` but it would only be for this one use case, and it would then make us loop over the meshes twice.

Reply:
Ah, thanks! 👍🏻 👍🏻